Controller/AuthorizeController.php

```diff
@@ -22,12 +22,12 @@
 use Symfony\Bundle\FrameworkBundle\Templating\EngineInterface;
 use Symfony\Component\DependencyInjection\ContainerAwareInterface;
 use Symfony\Component\DependencyInjection\ContainerInterface;
-use Symfony\Component\EventDispatcher\EventDispatcher;
 use Symfony\Component\Form\Form;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\RequestStack;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\HttpFoundation\Session\SessionInterface;
+use Symfony\Component\HttpKernel\Debug\TraceableEventDispatcher;
```

Comment:
This is incorrect, you don't typehint against a debug version.

Reply:
I totally don't understand why the initial version is not working and your comment. I just change this following error message, but I'm really not an expert.